### PR TITLE
Docs: Update whats-new-in-v6-7.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: Enhances readability and usability of release notes.

**Special notes for your reviewer**: Please refer to my questions in the comments.

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v6-7.md [hunk 1]
```diff
@@ -29,42 +29,48 @@ Grafana 6.7 comes with a lot of new features and enhancements:
...
 
 Allowing a low dashboard refresh interval can cause severe load on data sources and Grafana. Grafana v6.7 allows you to restrict the dashboard refresh interval so it cannot be set lower than a given interval. This provides a way for administrators to control dashboard refresh behavior on a global level.
 
-Refer to min_refresh_interval in [Configuration]({{< relref "../administration/configuration/#min-refresh-interval" >}}) for further information and how to enable this.
+Refer to min_refresh_interval in [Configuration]({{< relref "../administration/configuration/#min-refresh-interval" >}}) for more information and how to enable this feature.
+
+### Stackdriver project selector
 
-### Stackdriver Project Selector
 A Stackdriver data source in Grafana is configured for one service account only. That service account is always associated with a default project in Google Cloud Platform (GCP). Depending on your setup in GCP, the service account might be granted access to more projects than just the default project.
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v6-7.md]: What documentation was updated for this feature? Please link.

File: docs/sources/guides/whats-new-in-v6-7.md [hunk 2]
```diff
@@ -74,13 +80,17 @@ rendering_timeout = 10s
 concurrent_render_limit = 10
 ```
 
-### Export dashboard as pdf
-This feature allows you to export the dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
+### Export dashboard as PDF
+
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v6-7.md]: What documentation was updated for this feature? Please link.

File: docs/sources/guides/whats-new-in-v6-7.md [hunk 3]
```diff
@@ -74,13 +80,17 @@ rendering_timeout = 10s
...
 You can now use either portrait or landscape mode in your reports. Portrait will render three panels per page and landscape two.
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="1024px" caption="New report" >}}
 
+[Reporting]({{< relref "../enterprise/reporting/" >}}) has been updated as a result of this change.
+
 ### Azure OAuth Team Sync support
-When setting up OAuth with Microsoft Azure AD it’s now possible to sync Azure groups with Teams in Grafana.
-[Learn more]({{< relref "../auth/azuread/#team-sync-enterprise-only" >}}).
+When setting up OAuth with Microsoft Azure AD, you can now sync Azure groups with Teams in Grafana.
+Learn more in [Azure AD OAuth2 authentication]({{< relref "../auth/azuread/#team-sync-enterprise-only" >}}).
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v6-7.md]: Is this the right place to link to? Seems like the Team sync topic might make more sense.

**Comment by oddlittlebird** [comment]
If possible, use a standard format for the documentation paragraph. The link text should always be the name of the topic or section being linked to rather than something vague like "here" or "documentation."

File: docs/sources/guides/whats-new-in-v6-7.md (hunk 2 above)
Comment (jongyllen) [reply docs/sources/guides/whats-new-in-v6-7.md]: @alexanderzobnin is working on it, please link here when done

Comment (alexanderzobnin) [reply docs/sources/guides/whats-new-in-v6-7.md]: Pushed docs in #22746, so should be like 
```markdown
Learn more in [Export dashboard as PDF]({{< relref "../enterprise/export-pdf/" >}})
```

File: docs/sources/guides/whats-new-in-v6-7.md (hunk 3 above)
Comment (alexanderzobnin) [inline docs/sources/guides/whats-new-in-v6-7.md]: Yes, probably 
```suggestion
Learn more in [Team sync]({{< relref "../enterprise/team-sync/" >}}).
```

File: docs/sources/guides/whats-new-in-v6-7.md [hunk 4]
```diff
@@ -74,13 +80,17 @@ rendering_timeout = 10s
 concurrent_render_limit = 10
 ```
 
-### Export dashboard as pdf
-This feature allows you to export the dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
+### Export dashboard as PDF
+
+This feature allows you to export a dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
```
Comment (alexanderzobnin) [inline docs/sources/guides/whats-new-in-v6-7.md]:
```suggestion
This feature allows you to export a dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document. Learn more in [Export dashboard as PDF]({{< relref "../enterprise/export-pdf/" >}}).
```

**Review by jongyllen — APPROVED** [review]
Thanks!

**Pull request merged**